### PR TITLE
Added KilopoundsForcePerFoot (kipf/ft) imperial variant to ForcePerLength unit

### PR DESCRIPTION
**Pull request by dschuermans**
Added Kilopounds-force per foot imperial variant (kipf/ft)

### CODE REVIEW

File: UnitsNet.Tests/CustomCode/ForcePerLengthTests.cs [hunk 1]
```diff
@@ -27,6 +27,7 @@ public class ForcePerLengthTests : ForcePerLengthTestsBase
         protected override double DecinewtonsPerMeterInOneNewtonPerMeter => 1E1;
         protected override double KilogramsForcePerMeterInOneNewtonPerMeter => 0.101972;
         protected override double KilonewtonsPerMeterInOneNewtonPerMeter => 1E-3;
+        protected override double KilopoundsForcePerFootInOneNewtonPerMeter => 6.8521766E-5;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/ForcePerLengthTests.cs]: Verified.
https://www.cadforum.cz/cadforum_en/units.asp?u=forcePerLength

File: Common/UnitDefinitions/ForcePerLength.json [hunk 2]
```diff
@@ -83,6 +83,18 @@
...
+    },
+    {
+      "SingularName": "KilopoundForcePerFoot",
+      "PluralName": "KilopoundsForcePerFoot",
+      "FromUnitToBaseFunc": "x*14593.90292",
+      "FromBaseToUnitFunc": "x/14593.90292",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "kipf/ft" ]
```
Comment (angularsen) [inline Common/UnitDefinitions/ForcePerLength.json]: Isn't the abbreviation `kip/ft` ? Should we maybe add both, and if so, which should be the first array item used by ToString() by default?
Ref: https://www.cadforum.cz/cadforum_en/units.asp?u=forcePerLength

Comment (angularsen) [reply Common/UnitDefinitions/ForcePerLength.json]: Same goes for the now-merged https://github.com/angularsen/UnitsNet/pull/747.

Comment (angularsen) [reply Common/UnitDefinitions/ForcePerLength.json]: Another ref: https://en.wikipedia.org/wiki/Kip_(unit)

Comment (dschuermans) [reply Common/UnitDefinitions/ForcePerLength.json]: Well to be honest when I started on the implementation for imperial support in our module, I never really understood what the correct abbreviation was or should be.
We are using the abbreviation "k/ft" in our module.
Some places mention kipf/ft, other kip/ft.

For KilopoundsForce for example, we are using simply "k" and not kipf or kip.
For all i care, we can add all the abbreviations, or add them afterwards if someone needs them

candidates:

1.) kip/ft
2.) kipf/ft
3.) k/ft

Comment (dschuermans) [reply Common/UnitDefinitions/ForcePerLength.json]: Okay, mind if i update Force along too in this PR? or should I create a separate PR for that?

Comment (dschuermans) [reply Common/UnitDefinitions/ForcePerLength.json]: I'll do it separately

Comment (dschuermans) [reply Common/UnitDefinitions/ForcePerLength.json]: Done in #751

**Comment by angularsen** [comment]
[Release UnitsNet/4.45.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.45.0)

**Pull request merged**